.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,66 @@
+name: "PHPUnit tests"
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  phpunit:
+    name: "PHPUnit tests"
+
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        dependencies:
+          - "lowest"
+          - "highest"
+          - "locked"
+        php-version:
+          - "7.3"
+          - "7.4"
+          - "8.0"
+        operating-system:
+          - "ubuntu-latest"
+          - "windows-latest"
+          - "macos-latest"
+        include:
+          - php-version: "8.0"
+            dependencies: "highest"
+            operating-system: "ubuntu-latest"
```

Comment:
Isn't this a duplicate?

Reply:
715cf5d77eccf880ea002e2d927012f1c5216052